apps/src/lib/node/ledger/shims/abcipp_shim.rs

```diff
@@ -131,12 +131,23 @@ impl AbcippShim {
                     Ok(Resp::BeginBlock(Default::default()))
                 }
                 #[cfg(not(feature = "abcipp"))]
-                Req::DeliverTx(_) => Ok(Resp::DeliverTx(Default::default())),
+                Req::DeliverTx(tx) => {
+                    self.delivered_txs.push(tx.tx);
+                    Ok(Resp::DeliverTx(Default::default()))
+                }
                 #[cfg(not(feature = "abcipp"))]
                 Req::EndBlock(_) => {
-                    use namada::types::storage::BlockHash;
-                    let mut txs = vec![];
-                    std::mem::swap(&mut txs, &mut self.processed_txs);
+                    let (processing_results, _) =
+                        self.service.process_proposed_txs(&self.delivered_txs);
```

Comment:
This is the only sort of irrelevant line that should differ between `eth-bridge-integration` and `v0.8.1` now - in `eth-bridge-integration` we need to return a `DigestCounters` as part of `process_txs`/`process_proposed_txs`